src/main/java/com/michelin/kafkactl/command/Delete.java

```diff
@@ -148,8 +148,8 @@ public static class ByName {
         @Parameters(index = "0", description = "Resource type.", arity = "1")
         public String resourceType;
 
-        @Parameters(index = "1", description = "Resource name.", arity = "1")
-        public String name;
+        @Parameters(index = "1", description = "Resource name or wildcard matching resource names.", arity = "1")
+        public String resourceName;
```

Comment:
Outside of this PR, I am thinking that a `--execute` (just like we have in Reset-Password https://github.com/michelin/kafkactl/blob/b704582bf91abd575787bd0a437ebd4038ac6f5b/src/main/java/com/michelin/kafkactl/command/ResetPassword.java#L38) can be necessary if the resourceName contains any `*` to avoid unintentional `kafkactl delete topics *`

Reply:
Good idea, we can avoid some unwanted deletion.
So we would have:
- `kafkactl delete topics *` displaying a message like `You are about to potentially delete multiple resources with wildcard. Use --dry-run option to see the resources that will be deleted. Use --execute option to confirm the deletion.`
- `kafkactl delete topics * --dry-run` using the deletion endpoint with dryrun and displaying the list of resources that will be deleted.
- `kafkactl delete topics * --execute` using the deletion endpoint without dryrun and displaying the list of deleted resources.